libs/watt/src/lib/components/data/watt-data-table.component.ts

```diff
@@ -0,0 +1,124 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Component, ContentChild, Input, ViewEncapsulation, inject } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+import {
+  VaterFlexComponent,
+  VaterSpacerComponent,
+  VaterStackComponent,
+  VaterUtilityDirective,
+} from '../vater';
+
+import { WattCardComponent } from '../card';
+import { WattSearchComponent } from '../search';
+import { WattTableComponent } from '../table';
+import { WattPaginatorComponent } from '../paginator';
+import { WattEmptyStateComponent } from '../empty-state';
+
+import { WattDataIntlService } from './watt-data-intl.service';
+
+@Component({
+  selector: 'watt-data-table',
+  standalone: true,
+  imports: [
+    CommonModule,
+    VaterFlexComponent,
+    VaterSpacerComponent,
+    VaterStackComponent,
+    VaterUtilityDirective,
+    WattCardComponent,
+    WattEmptyStateComponent,
+    WattPaginatorComponent,
+    WattSearchComponent,
+  ],
+  encapsulation: ViewEncapsulation.None,
+  styles: [
+    `
+      watt-data-table h3,
+      watt-data-table h4 {
+        line-height: 44px;
```

Comment:
Question: So we have "headers" with different font sizes and potential other styles, since only the line height and height are styled here?

Reply:
Yes, I based that decision on the designs in Figma, which uses `h3` on the "main" view and `h4` in the drawer:
https://www.figma.com/file/ipM0Z3VUyrk0SQTvAfmfnw/Wholesale?type=design&node-id=378-64675&mode=design&t=WClfDU1vEJ3MVmW8-0

Reply:
Ah yes, of course that makes sense!